tmt/base.py

```diff
@@ -1492,6 +1497,17 @@ def export(self, format_='yaml'):
             raise tmt.utils.GeneralError(
                 f"Invalid plan export format '{format_}'.")
 
+    @property
+    def imports_plan(self) -> bool:
+        return self._import_plan_node is not None
+
+    @property
+    def imported_plan(self) -> Optional['Plan']:
+        if not self.imports_plan:
+            return None
+        node = fmf.Tree.node(self._import_plan_node)
+        return Plan(node=node, run=self.parent, logger=self)
```

Comment:
I'd suggest caching the `Plan` instance, `self._imported_plan` to complement `self._imported_plan_node`.

Reply:
Caching is added.